packages/components/src/presentation-components/instance-filter-builder/InstanceFilterConverter.ts

```diff
@@ -122,6 +122,11 @@ function createComparison(propertyName: string, type: string, alias: string, ope
     case "number":
       valueExpression = value.toString();
       break;
+    case "object":
+      if (isPoint2d(value)) {
+        return createPointComparision(value, operatorExpression, propertyAccessor);
+      }
+      break;
```

Comment:
I think this should be handled outside `switch` statement like `navigation` and `dateTime` properties.

Reply:
fixed